packages/@aws-cdk-testing/framework-integ/test/aws-dynamodb/test/integ.dynamodb.contirubtor-insights-for-gsi.ts

```diff
@@ -0,0 +1,41 @@
+import { App, RemovalPolicy, Stack } from 'aws-cdk-lib';
+import { Attribute, AttributeType, Table } from 'aws-cdk-lib/aws-dynamodb';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+
+// CDK parameters
+const STACK_NAME = 'aws-cdk-dynamodb-contributor-insights-for-gis';
```

Comment:
What is the meaning of `gis`? If it means global secondary index, it is better to use `gsi`.

Reply:
As you pointed out, it was a mistake.
I have corrected it.​​​​​​​​​​​​​​​​